helmfile/charts/prometheus-alerts/templates/rules/backup-status.yaml

```diff
@@ -0,0 +1,144 @@
+        description: The job daily backup job velero-backup have failed over 48 hours.
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-velerobackup
+        summary: The daily backup job velero-backup have failed over 48 hours.
+      expr: |-
+        (
+          (time() - velero_backup_last_successful_timestamp{schedule="velero-daily-backup", cluster=~".*"}) / 60 / 60 > 48
+        )
+      for: 1h
+      labels:
+        severity: warning
+    - alert: ElasticsearchBackupHaveFailed12Hours
+      annotations:
+        description: The job daily backup job elasticsearch-backup have failed over 12 hours.
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-elasticsearchbackup
+        summary: The daily backup job elasticsearch-backup have failed over 12 hours.
+      expr: |-
+        (
+          min((time()-kube_job_status_completion_time{job_name=~"opensearch-backup-.*", cluster=~".*"})/3600) > 12
+        )
+      for: 1h
+      labels:
+        severity: warning
+    - alert: ElasticsearchBackupHaveFailed24Hours
+      annotations:
+        description: The job daily backup job elasticsearch-backup have failed over 24 hours.
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-elasticsearchbackup
+        summary: The daily backup job elasticsearch-backup have failed over 24 hours.
+      expr: |-
+        (
+          min((time()-kube_job_status_completion_time{job_name=~"opensearch-backup-.*", cluster=~".*"})/3600) > 24
+        )
+      for: 1h
+      labels:
+        severity: warning
+    - alert: ElasticsearchSnapshotHaveFailed12Hours
+      annotations:
+        description: The job daily backup job {{`{{`}} $labels.repository {{`}}`}} have failed over 12 hours.
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-elasticsearchsnap
+        summary: The daily backup job elasticsearch snapshot have failed over 12 hours.
+      expr: |-
+        (
+          (time()-elasticsearch_snapshot_stats_snapshot_start_time_timestamp{state="SUCCESS", cluster=~".*"})/3600 > 12
+        or
+          (time()-elasticsearch_snapshot_stats_snapshot_start_time_timestamp{state="PARTIAL", cluster=~".*"})/3600 > 12
+        )
+      for: 1h
+      labels:
+        severity: warning
+    - alert: ElasticsearchSnapshotHaveFailed24Hours
+      annotations:
+        description: The job daily backup job {{`{{`}} $labels.repository {{`}}`}} have failed over 24 hours.
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-elasticsearchsnap
+        summary: The daily backup job elasticsearch snapshot have failed over 24 hours.
+      expr: |-
+        (
+          (time()-elasticsearch_snapshot_stats_snapshot_start_time_timestamp{state="SUCCESS", cluster=~".*"})/3600 > 24
+        or
+          (time()-elasticsearch_snapshot_stats_snapshot_start_time_timestamp{state="PARTIAL", cluster=~".*"})/3600 > 24
+        )
+      for: 1h
+      labels:
+        severity: warning
```

Comment:
I know that the 12/24 hours here are based on the dashboard in grafana. But I think I would rather have them as 24/48 hours so that all of the alerts are aligned.